erts/emulator/beam/beam_emu.c

```diff
@@ -1287,6 +1287,8 @@ void process_main(Eterm * x_reg_array, FloatDef* f_reg_array)
      */
     int neg_o_reds = 0;
 
+    ErtsCodeMFA *mfa_bif = NULL;
```

Comment:
Introducing a new variable at the top level could be surprisingly expensive, especially for Clang. If at all possible, try to limit the scope of temporary variables by putting them in a block.